config/default/GUIDE.md

```diff
@@ -6,7 +6,7 @@ Detailed tutorials on how to build apps from scratch with the Core Framework wil
 
 ## Modifying the Reference Application
 
-You should have the [Core Framework running](../../INSTALL.md), have installed [medic-conf](https://github.com/medic/medic-conf) and have checked out or otherwise downloaded the [Reference Application](./).
+You should have the [Core Framework running](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/#1-install-the-core-framework), have installed [medic-conf](https://github.com/medic/medic-conf) and have checked out or otherwise downloaded the [Reference Application](./).
```

Comment:
Same as above:

```suggestion
You should have the [Core Framework running](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/), have installed [medic-conf](https://github.com/medic/medic-conf) and have checked out or otherwise downloaded the [Reference Application](./).
```